### DISCUSSION TITLE
Appearance Changes

### DISCUSSION THREAD

**Discussion by GoldenGideon** [YukiGasai/obsidian-google-calendar#272] [Ideas]
I just switched over from a different calendar plugin because this one had some features I sorely needed.  Thus far I am enjoying it, but the appearance is throwing me off.  Would it be possible to get rid of the shadows that are around each event block and add the event times to the block?  I'd like to see my event times without having to click on each event.  It could be a toggleable feature perhaps, so that people could have it both ways :)

**Comment by r-macewen** [comment]
I also find that 30 minute blocks are too narrow. The text size is bigger than that, and, multiple meetings in series overlap each other and you can't read anything. The height of each 30min block needs to be higher
